src/io/mod.rs

```diff
@@ -67,11 +68,3 @@ pub trait KVStore {
 	/// Will return an empty list if the `namespace` is unknown.
 	fn list(&self, namespace: &str) -> std::io::Result<Vec<String>>;
```

Comment:
i think this should be done out of bounds, 
an application needs to know its namespaces for e.g "channel_store", "peer_store" etc. instead of list_namespaces.